api/interestr/api/serializers.py

```diff
@@ -233,3 +233,47 @@ class Meta:
         model = auth_models.User
         fields = ('id', 'username', 'email', 'joined_groups', 'data_templates',
                   'posts', 'profilepage', 'votes', 'moderated_groups', )
+
+class AnnotationSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = core_models.Annotation
+        exclude = ( )
+
+    def to_representation(self, instance):
+        data = super(AnnotationSerializer, self).to_representation(instance)
+        json_str = {"@context": "http://www.w3.org/ns/anno.jsonld", 
+            "id": "http://interestr.com/annotations/" + str(instance.anno_id), 
+            "type": "Annotation",
+            "created": instance.created,
+            "creator": {
+            "id": "http://interestr.com/profile/" + str(instance.user.profile.id),
+            "type": "Person",
+            "name":  instance.user.profile.name + " " +  str(instance.user.profile.surname),
+            "nickname": instance.user.username ,
+            "email": instance.user.email
+                },
+
+            "bodyValue": instance.text,
+            "target": {
+            "source":  str(instance.target),
+                "type": str(instance.annotype),
+             "selector":  str(instance.selector)           }
+
+            }
+        return json_str
+
+    def to_internal_value(self, data):
+        
+        profile_id = data["creator"]["id"].strip("http://interestr.com/profile/")
```

Comment:
Our website is not at `interestr.com`, yet. For now, we may just strip anything before the `/profile/`.